jekyll/assets/js/sidebar.js

```diff
@@ -1,89 +1,91 @@
+    var mobileSidebarDefault = mobileSidebar.querySelector('[data-id="welcome"]');
+    var mobileSidebarDisplay = mobileSidebar.querySelector('.mobile-sidebar');
 
-  // get hash, if it exists
-  if (window.location.hash && window.location.hash.indexOf('section') > -1) {
-    var section = getUrlVars(window.location.hash);
-    localStorage.sidenavActive = section['section']
-  }
+    // get hash, if it exists
+    if (window.location.hash && window.location.hash.indexOf('section') > -1) {
+      var section = getUrlVars(window.location.hash);
+      localStorage.sidenavActive = section['section']
+    }
 
-  if (localStorage.sidenavActive) {
-    // fullscreen sidenav expansion
-    function sidenavAutoExpand (parent) {
-      var element = parent.querySelector('[data-section=' + localStorage.sidenavActive + ']');
-      if (element.classList.contains('closed')) {
-        element.classList.remove('closed');
-      }  
-    };
+    if (localStorage.sidenavActive) {
+      // fullscreen sidenav expansion
+      function sidenavAutoExpand (parent) {
+        var element = parent.querySelector('[data-section=' + localStorage.sidenavActive + ']');
+        if (element.classList.contains('closed')) {
+          element.classList.remove('closed');
+        }  
+      };
 
-    sidenavAutoExpand(sidebar);
-    sidenavAutoExpand(mobileSidebar);
+      sidenavAutoExpand(sidebar);
+      sidenavAutoExpand(mobileSidebar);
 
-    // for mobile sidebar, if sidebar is set, display proper item
-    var mobileCurrentElement = mobileSidebar.querySelector('[data-id=' + localStorage.sidenavActive + ']');
-    if (mobileCurrentElement.classList.contains('hidden')) {
-      mobileCurrentElement.classList.remove('hidden');
-      mobileSidebarDefault.classList.add('hidden');
+      // for mobile sidebar, if sidebar is set, display proper item
+      var mobileCurrentElement = mobileSidebar.querySelector('[data-id=' + localStorage.sidenavActive + ']');
+      if (mobileCurrentElement.classList.contains('hidden')) {
+        mobileCurrentElement.classList.remove('hidden');
+        mobileSidebarDefault.classList.add('hidden');
+      }
     }
-  }
 
-  // Show/hide mobile sidebar
-  mobileSidebarCurrent.addEventListener('click', function () {
-    if(mobileSidebarDisplay.classList.contains('hidden')) {
-      mobileSidebarDisplay.classList.remove('hidden');
-    } else {
-      mobileSidebarDisplay.classList.add('hidden');
-    }
-  });
-
-  function setSidebar () {
-    // if footer is in frame, removed fixed style (otherwise add it, if it doesn't exist)
-    if ((footer.getBoundingClientRect().top - window.innerHeight) <= 0 && footer.getBoundingClientRect().top >= window.innerHeight) {
-      if (sidebar.classList.contains('fixed')) {
-        sidebar.classList.remove('fixed');
+    // Show/hide mobile sidebar
+    mobileSidebarCurrent.addEventListener('click', function () {
+      if(mobileSidebarDisplay.classList.contains('hidden')) {
```

Comment:
Could we use the `hidden` class for the template instead of `display: none`? It would be nice to be consistent between those, but I'm not sure if instantsearch does it's own thing 😆